### PR TITLE
Added task and reference related to the grafana workload

### PR DESCRIPTION
**Pull request by tbazant**

### CODE REVIEW

File: xml/task-run-grafana-with-podman.xml [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+      <step>
+        <para>
+          Open the &grafana; Web page that is running on port 3000 on the
+          &alpshort; host where the &grafana; workload is running, for example:
+        </para>
+<screen>&prompt.user;firefox http://<replaceable>ALP_HOST_IP_ADDRESS</replaceable>:3000</screen>
+      </step>
+      <step>
+        <para>
+          Log in to &grafana;. The default user name and password is both set
```
Comment (dariavladykina) [inline xml/task-run-grafana-with-podman.xml:135]:
```suggestion
          Log in to &grafana;. The default user name and password are both set
```

File: xml/task-run-grafana-with-podman.xml [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+          to <literal>admin</literal>. After logging in, enter a new password.
+        </para>
+      </step>
+      <step>
+        <para>
+          Add the &prometheus; data source provided by the client. In the left
+          panel, hover you mouse over the gear icon and select <guimenu>Data
```
Comment (dariavladykina) [inline xml/task-run-grafana-with-podman.xml:142]:
```suggestion
          panel, hover your mouse over the gear icon and select <guimenu>Data
```

File: xml/task-run-grafana-with-podman.xml [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+          Confirm with <guimenu>Save &amp; test</guimenu>
+        </para>
+      </step>
+      <step>
+        <para>
+          Create a dashboard based on &prometheus; data. Hover your mouse over
+          the plus sign in the left panel and select <guimenu>Import</guimenu>.
+        </para>
+        <figure>
+          <title>Creating &grafana; dashboard</title>
```
Comment (dariavladykina) [inline xml/task-run-grafana-with-podman.xml:189]:
```suggestion
          <title>Creating a &grafana; dashboard</title>
```

File: xml/task-run-grafana-with-podman.xml [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+            </imageobject>
+            <imageobject role="html">
+              <imagedata fileref="alp-grafana-prometheus-url.png" width="75%"/>
+            </imageobject>
+            <textobject role="description"><phrase>&prometheus; URL configuration in &grafana;</phrase>
+            </textobject>
+          </mediaobject>
+        </figure>
+        <para>
+          Confirm with <guimenu>Save &amp; test</guimenu>
```
Comment (dariavladykina) [inline xml/task-run-grafana-with-podman.xml:180]:
```suggestion
          Confirm with <guimenu>Save &amp; Test</guimenu>
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some suggestions here. Thanks!

File: xml/task-run-grafana-with-podman.xml (hunk 4 above)
Comment (tbazant) [reply xml/task-run-grafana-with-podman.xml:180]: @dariavladykina  i copied exactly the string from the GUI, i think we should not modify what is in the application, or should we?

Comment (dariavladykina) [reply xml/task-run-grafana-with-podman.xml:180]: Interesting, as I found screenshots from GUI where 'Test' was capitalized. Probably they are outdated, so let's keep the original.

**Pull request merged**